spec/support/config/devise.rb

```diff
@@ -1,13 +1,4 @@
 RSpec.configure do |config|
-  config.include Devise::TestHelpers, type: :controller
-  config.include Devise::TestHelpers, type: :view_component
-
-  module DeviseTestHelpersWithScope
-    def sign_in(user)
-      super(:user, user)
-    end
```

Comment:
Let's update this one method to pass the scope option instead of having to change all current and future usages of `sign_in`.

Reply:
I disagree strongly, actually.

By using the same name as the built-in method, we create confusion because the arguments are different. I personally don't mind all the use of `sign_in` in the specs, because this is formal API surface of devise, and [it is the recommended approach when testing](https://github.com/plataformatec/devise/wiki/How-To:-Sign-in-from-a-controller).

Arguably it could be more DRY, but I think the added layer of indirection hurts more than it helps.

Also: yes, surprisingly this version Just Worked, which is pretty cool!

Reply:
Alright then. I see your point. Let's go with the explicit/recommended approach then.